app/javascript/onboarding/components/ProfileForm.jsx

```diff
@@ -10,22 +10,24 @@ class ProfileForm extends Component {
     super(props);
 
     this.handleChange = this.handleChange.bind(this);
+    this.handleColorPickerChange = this.handleColorPickerChange.bind(this);
     this.onSubmit = this.onSubmit.bind(this);
     this.user = userData();
-
     this.state = {
-      formValues: {
-        summary: '',
-        location: '',
-        employment_title: '',
-        employer_name: '',
-      },
-      last_onboarding_page: 'v2: personal info form',
+      groups: [],
+      formValues: {},
       canSkip: true,
+      last_onboarding_page: 'v2: personal info form',
     };
   }
 
   componentDidMount() {
+    fetch('/profile_field_groups?onboarding=true')
```

Comment:
refactor (blocking): Use the [request](https://github.com/forem/forem/blob/master/app/javascript/utilities/http/request.js#L23) utility function instead via `import { request } from '@utilities/http`. We added the `request` function a little while back after we saw that folks were doing the same things over and over.

With the `request` function, there is no need to pass the headers that are currently specified as they are default headers baked in. As well, there is no need to do a `JSON.stringify` for the body. This is handled within `request`.

Reply:
Thanks @nickytonline I missed a lot since last working on the frontend 😁. I've moved the code over, this is nicer!